### ISSUE TITLE
Status bar scanning messages don't go away

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#2687]
https://github.com/UniversalMediaServer/UniversalMediaServer/issues/2678 didn't resolve the problem. I still see "Scanning Folder:" messages when nothing is happening
@valib

**Comment by SubJunk** [comment]
@valib I hope you're doing well, and hope you have been released

**Comment by valib** [comment]
@SubJunk not exactly. I am now on the rehabilitation department to let me at least to go to the toilet and so on.

**Comment by SubJunk** [comment]
@valib damn that's a long time :( my thoughts are with you

**Comment by SubJunk** [comment]
@valib how are you?

**Comment by valib** [comment]
Thanks for the interest. I am at home, my wife takes care about me and I am exersizing the leg to be able to walk without a stick.

**Comment by SubJunk** [comment]
@valib I'm relieved you're at home now at least. Is it getting better? Is the exercise painful?

**Comment by valib** [comment]
Yes to be at home is much better and yes exercising is a little bit painful. You know when you not use your extremities the tendons are shortened and need to extend them. Of course with some pain. 🙂

**Comment by Nadahar** [comment]
@valib It sounds like you've had a really bad situation. It's a bit strange to take in that a "small accident" can give so serious consequences. It's certainly a reminder that we should think twice before taking unnecessary or be unfocused.

I hope you will make a full recovery and that you won't ever be this unlucky again. It's so easy to forget that we're getting older, when I was 20 I could fall pretty much any way I wanted without getting a serious injury. When I was 17-18, I fell a lot while driving motorcycle in the winter (it was the only vehicle I had, so I used it in all kind of weather), and I was so used to it not having any consequences that I didn't really have any respect for falling at all. As a consequence I took chances and fell much more often than "necessary", even at speeds of 70-80 km/h.

The tricky thing, for me at least, is to get my brain to understand that those rules no longer apply :wink:

**Comment by SubJunk** [comment]
@valib that's good news you are at that stage, please keep us updated on your recovery! We have collaborated for over a decade now so you feel a bit like family, I want you to do well.

**Comment by SubJunk** [comment]
@valib how are you doing?

**Comment by valib** [comment]
@SubJunk thanks I am fine. The doctor allowed me to start walking so I am trying. My walk looks like the penguin but it is going better and better.

**Comment by SubJunk** [comment]
@valib that's so great! Now that there is a new Batman movie franchise you could audition to be The Penguin?

**Comment by SubJunk** [comment]
This got fixed along the way
